index.html

```diff
@@ -695,9 +711,10 @@ <h3>Proof Chains</h3>
 A proof chain is useful when the same data needs to be signed by
 multiple entities and the order of when the proofs occurred matters,
```

Comment:
Ties into the above — outer/later signing must be checked before inner/earlier signing, and the "required order" will be the reverse of outermost-to-innermost order.

Reply:
See response here: https://github.com/w3c/vc-data-integrity/pull/85/files#r1125499681

TL;DR: Which order we go in doesn't matter at this point in time.